app/controllers/people_skills/params.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module PeopleSkills
+  class Params
+
+    def initialize(params)
+      @params = params
+    end
+
+    def skills
+      return [] if @params[:skill_id].nil? || @params[:skill_id].blank?
+
+      @params[:skill_id].select { |skill| skill.present? }
+    end
+
+    def levels
+      @params[:level].presence
```

Comment:
A method called `levels` should in my world return levels, not check for their presence. How about `levels_present` ?